spec/controllers/api/v1/organizations_controller_spec.rb

```diff
@@ -0,0 +1,135 @@
+require 'spec_helper'
+
+describe Api::V1::OrganizationsController, type: :controller do
+  let(:authorized_user) { create(:user) }
+  let(:organization) { build(:organization, listed_at: Time.now, owner: authorized_user) }
+  let(:unlisted_organization) { build(:organization, listed_at: nil) }
+
+  let(:scopes) { %w(public organization) }
+
+  describe "when not logged in" do
+    describe "#index" do
+
+      it 'returns only the listed organization' do
```

Comment:
I added an extra expect line to make sure the non-logged in user can't see an unlisted org which is unowned by them. Same spec since it's testing the same thing, that only listed orgs are returned.